.github/workflows/e2e_tests.yml

```diff
@@ -189,3 +190,81 @@ jobs:
         with:
           message: "An E2E test failed on the upstream develop branch."
           webhook-url: ${{ secrets.BUILD_FAILURE_ROOM_WEBHOOK_URL }}
+  acceptance_test:
+    name: Acceptance tests
+    needs: build
+    runs-on:  ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-22.04]
+        suite:
+          - "should assign roles to users and change tag properties"
+          - "should check blog editor unable to publish duplicate blog post"
+          - "should create draft and delete draft blog post"
+          - "should publish blog post and delete published blog post"
```

Comment:
Hmm, what will be the final number of those?

Reply:
It will be equaling the same no. of total user stories that will be made .